templates/new_service/.github/workflows/build-nocache.yml

```diff
@@ -0,0 +1,45 @@
+name: Build No Cache
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: '30 12 * * 0'
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  build-no-cache:
+    env:
+      DOCKER_IMAGE: #DOCKER_REPOSITORY#
```

Comment:
Can we rename to DOCKER_REPOSITORY for clarity?

Reply:
done